crates/iota-types/src/digests.rs

```diff
@@ -1074,3 +1074,127 @@ mod test {
         assert_eq!(chain_id, None);
     }
 }
+
+/// MoveAuthenticationDigest is the hash (digest) of the `GenericSignature``
+/// payload when the transaction uses a MoveAuthenticator as its signature
```

Comment:
```suggestion
/// payload when the transaction uses a `MoveAuthenticator` as its signature
```
Since GenericSignature is quoted, it is better to be consistent in the documentation.

Reply:
Done